var/spack/repos/builtin/packages/sim-common/package.py

```diff
@@ -1,39 +1,27 @@
-##############################################################################
-# Copyright (c) 2013-2018, Lawrence Livermore National Security, LLC.
-# Produced at the Lawrence Livermore National Laboratory.
-#
-# This file is part of Spack.
-# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
-# LLNL-CODE-647188
-#
-# For details, see https://github.com/spack/spack
-# Please also see the NOTICE and LICENSE files for our notice and the LGPL.
-#
-# This program is free software; you can redistribute it and/or modify
-# it under the terms of the GNU Lesser General Public License (as
-# published by the Free Software Foundation) version 2.1, February 1999.
-#
-# This program is distributed in the hope that it will be useful, but
-# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
-# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
-# conditions of the GNU Lesser General Public License for more details.
-#
-# You should have received a copy of the GNU Lesser General Public
-# License along with this program; if not, write to the Free Software
-# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
-##############################################################################
+# Copyright 2013-2018 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
 from spack import *
 from spack.pkg.builtin.neurodamus_base import NeurodamusBase
-import os
+from contextlib import contextmanager
 import shutil
+import os
 import sys
-from contextlib import contextmanager
 
 
-class Neurodamus(NeurodamusBase):
-    """Package used for building special from NeurodamusBase package
+class SimCommon(Package):
```

Comment:
my idea was to avoid naming changes again later. Since I expected models to be independent of neurodamus, calling it `neurodamus-cortex` could be misleading. But I also agree the name could have some allusion to neurodamus, I just couldnt come up with a better name. So I thought better to change from the beginning to something meaningful and teach users :)

Reply:
Conclusion: For the moment we use the naming neurodamus-xxx. In the future we consider that hard splitting.